src/lib/fabric-custom-shared.ts

```diff
@@ -0,0 +1,25 @@
+const CACHE_BUST = '%%CACHEBUSTER%%';
+const DapAssetsRoot = `https://s3-eu-west-1.amazonaws.com/adops-assets/dap-fabrics`;
+
+export const getDapAssetsPath = (DapAssetsFolder: string) =>
+	`${DapAssetsRoot}/${DapAssetsFolder}`;
+
+export const addTrackingPixel = (url: string) => {
+	const pixel = new Image();
+	pixel.src = url + CACHE_BUST;
+};
+
+// relative paths in the CSS need to be replaced with the absolute path
+export const replaceAssetLinks = (html: string, DapAssetsPath: string) => {
+	const re = /url\('\.\/(.*)'\)/g;
+	return html.replace(re, `url('${DapAssetsPath}/$1')`);
+};
+
+export const getTag = (DapAssetsFolder: string, DapAssetsPath: string) => {
+	if (DapAssetsFolder) {
+		return fetch(`${DapAssetsPath}/index.html`)
+			.then((res) => res.text())
+			.then((html) => replaceAssetLinks(html, DapAssetsPath));
+	}
+	return Promise.reject('No tag found');
+};
```

Comment:
```suggestion
export const addTrackingPixel = (url: string) => {
	const pixel = new Image();
	pixel.src = url + CACHE_BUST;
};

// relative paths in the CSS need to be replaced with the absolute path
export const replaceAssetLinks = (html: string, DapAssetsPath: string) => {
	const re = /url\('\.\/(.*)'\)/g;
	return html.replace(re, `url('${DapAssetsPath}/$1')`);
};

export const getTag = (DapAssetsFolder: string) => {
	if (DapAssetsFolder) {
		return fetch(`${DapAssetsRoot}/${DapAssetsFolder}/index.html`)
			.then((res) => res.text())
			.then((html) => replaceAssetLinks(html, DapAssetsPath));
	}
	return Promise.reject('No tag found');
};
```

Reply:
Thanks @Jakeii, are we sure we want to remove the DapAssetsPath parameter from the `getTag` function? 
This causes an error as we need it on line19 when calling replaceAssetLinks

Reply:
@Jakeii, are we sure we want to remove the `DapAssetsPath` parameter from the `getTag` function, as its causing an error. Needed as an argument on line 19 `.then((html) => replaceAssetLinks(html, DapAssetsPath));` and that value is not in scope in this module.

Reply:
Ah it'll need to be a local variable inside `getTag` then

Reply:
@Jakeii, wont need to receive it somehow from the `src/templates/ssr/fabric-custom-xl/index.ts`? as the variable is set as part of thew component template?

Reply:
`DapAssetsPath` is made from `${DapAssetsRoot}/${DapAssetsFolder}`, `DapAssetsRoot` is in this file, and `DapAssetsFolder` is available in `getTag` as an argument so can get rid of `getDapAssetsPath`

Reply:
Thanks @Jakeii didn't spot that! 👍